Lesson24/selcan1/index..html

```diff
@@ -0,0 +1,148 @@
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.2/css/all.min.css" integrity="sha512-SnH5WK+bZxgPHs44uWIX+LLJAJ9/2PkPKZ5QiAj6Ta86w+fsb2TkcmfRyVX3pBnMFcV7oQPJkl9QevSCWr3W6A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Document</title>
+    <style>
+        .kapsayici{
+            width: 100%;;
+            height:100vh;
+            background-color: rgb(60, 60, 104); 
+            display: flex; 
+            justify-content: center;  
+            align-items: center;   
+          }
+        .anaMenu{
+            border-radius: 25px;
+            background-color: white;
+            width: 50%;
+            height: 50%;
+            padding: 30px;
+            display: flex;
+            flex-direction: row;  
+         }
+         @media all and (max-width: 576px) {
+        .kapsayici{
+            background-color: white;
+        }
+         .anaMenu{
+            flex-direction:column-reverse;
+            height:100%;
+            width: 100%;
+         }
+      }
+         .sagEkran{
+            display: flex;
+            margin-left: 90px;
+            justify-content: center;
+         }
+        .sagEkran>img{
+            width: 90%;
+            height: 100%;
+            border-radius:20px;
+        }
+         h2{
+            font-size: 30px;
+            padding: 0 20px;
+         }
+         ul{
+            line-height: 30px;
+            padding-left: 0;
+         }
+         .solEkran li{
+            list-style-type: none;
+            padding-left:10px;  
+         }
+         ul .kutu{
+            display: flex;
+            align-items: flex-start;
+         } 
+         .fa-circle-check{
+         color: red;
+         }
+         .form-input{
+            width: 100%;
+            height: 35px;
+            border-radius: 5px;
+            margin-bottom: 20px;
+         }
+         button{
+            width:calc(100% + 10px);
+            height: 50px;
+            background-color: rgb(4, 4, 30);
+            color: white;
+            padding: 5px auto;
+            border-radius: 10px;
+            cursor: pointer; 
+         }
+         button:hover{
+            background-color: #590808;
+            transition: .5s;
+         }
+        .error {
+            color: #590808;
+            font-size: 12px;
+            margin-top: 5px;
+            margin-bottom: 10px;
+            display: flex; 
+            justify-content: flex-end;  
+        }
+        .sayfa2{
+         display: none;
+        }
+       
+
+       
+    </style>
+</head>
+<body>
+    <div class="kapsayici">
+        
+       <div class="anaMenu">
+        <div class="solEkran">
+            <h2>Stay Updated!</h2>
+            <p>Join 60.0004 product managers receiving monthly updates on:</p>
+           <ul>
+               <div class="kutu"><i class="fa-solid fa-circle-check"></i><li>Product discovery and building what matters</li></div>  
```

Comment:
The first child of the list element should be a list item. Refactor code so that it looks like this instead:
```
<li>
  <i class="fa-solid fa-circle-check"></i>
  <span>Product discovery and building what matters</span>
</li>
```

Do that for all list items and update CSS so that each list item is a flex container, then center the content vertically (right now icon and text are not aligned).